### ISSUE TITLE
Add example script that generates gif shown on the README page

### ISSUE THREAD

**Issue by GenevieveBuckley** [napari/napari-animation#165]
In the same vein as https://github.com/napari/napari-animation/issues/163, it would be nice if the script that generated the gif example shown on the main [README](https://github.com/napari/napari-animation/blob/main/README.md) page was included as [one of the examples here](https://github.com/napari/napari-animation/tree/main/examples)

I think @alisterburt was the person who made this example, perhaps he still has the code somewhere?

![README gif](https://user-images.githubusercontent.com/7307488/196110138-6c4663b1-67b2-4c79-97b7-57b706d1d49c.gif)

**Comment by alisterburt** [comment]
agree! unfortunately this was a quick bodge and I don't have the file any more - can think about using a new gif with code or remaking
